packages/app/src/pages/StakeDashboard.tsx

```diff
@@ -114,6 +126,7 @@ export const StakeDashboard: FC = () => {
           </div>
         )}
         <StakeForm
```

Comment:
Can you refactor it to have a StakeForm and a UnstakeForm which are shown depending on isStakeSelected, this way we can clearly separate the business logic for staking and unstaking and don't have add logic in event handlers if it's this or that state.

Reply:
Done.